### PR TITLE
[DOCS] Threat Intel Card uses Fleet TI integrations

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses https://github.com/elastic/security-docs/issues/1431 and https://github.com/elastic/security-docs/issues/1432. 

- Preview for the revised "Threat intelligence" section is [here](https://security-docs_1455.docs-preview.app.elstc.co/guide/en/security/master/es-ui-overview.html). (Go to Overview -> Threat intelligence.)
- Preview for the integration enablement steps is available in [here](https://security-docs_1455.docs-preview.app.elstc.co/guide/en/security/master/es-threat-intel-integrations.html).

### CODE REVIEW

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
+[[es-threat-intel-integrations]]
+= Enable threat intelligence integrations
+
+The Threat Intelligence view gives you a streamlined way to collect threat intelligence data that you can use for threat detection and matching. You can connect to threat intelligence sources using an <<agent-ti-integration, {agent} integration>>, the <<ti-mod-integration, Threat Intel module>>, or a <<custom-ti-integration, custom integration>>.
+
+NOTE: The warning message about remaining threat intelligence sources is removed once you’ve enabled all available {elastic-sec} threat intelligence integrations.
```
Comment (joepeeples) [inline docs/getting-started/threat-intel-integrations.asciidoc:6]: Unclear what warning message this refers to. I think it's talking about the message shown int the screenshot, but the phrasing throws me off: the message is not really a *warning,* and it doesn't refer to "remaining sources," just disabled ones.

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+[role="screenshot"]
+image::images/threat-intelligence-view.png[width=90%][height=90%][Shows the Threat Intelligence view on the Overview page]
+
+[float]
+[[agent-ti-integration]]
+== Enable an {agent} integration
+
+To set up an {agent} integration:
+
+. Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet} managed {agent}] on the servers you want to monitor.
```
Comment (joepeeples) [inline docs/getting-started/threat-intel-integrations.asciidoc:17]:
```suggestion
. Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] on the servers you want to monitor.
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
+
+[float]
+[[custom-ti-integration]]
+== Enable a custom integration
+
+To enable a custom integration:
+
+. Set up a way to <<ingest-data, ingest data>> into your system.
+. Update the `securitySolution:defaultThreatIndex` <<update-threat-intel-indices, advanced setting>> by adding the appropriate index pattern name after the default {fleet} threat intelligence index pattern (`logs-ti*`), for example, `custom-ti-index*`.
```
Comment (joepeeples) [inline docs/getting-started/threat-intel-integrations.asciidoc:44]: The example is a little confusing to me. If the user has to add something to the end of the default index pattern, which is `logs-ti*`, shouldn't the example _include_ that default pattern plus something else?

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+== Enable a Threat Intel module integration
+
+To set up an integration through the Threat Intel module:
+
+. Set up the {filebeat-ref}/filebeat-installation-configuration.html[{filebeat} agent] and enable the Threat Intel module.
++
+NOTE: For more information about enabling available threat intelligence filesets, refer to {filebeat-ref}/filebeat-module-threatintel.html[Threat Intel module].
+
+. Update the `securitySolution:defaultThreatIndex` <<update-threat-intel-indices, advanced setting>> by adding the appropriate index pattern name after the default {fleet} threat intelligence index pattern (`logs-ti*`):
+.. If you're using {filebeat} version 8.x _only_, add the appropriate {filebeat} threat intelligence index pattern, for example, `filebeat-8*` or `filebeat*`.
```
Comment (benironside) [inline docs/getting-started/threat-intel-integrations.asciidoc:33]:
```suggestion
.. If you're _only_ using {filebeat} version 8.x, add the appropriate {filebeat} threat intelligence index pattern, for example, `filebeat-8*` or `filebeat*`.
```

**Review by benironside — APPROVED** [review]
LGTM, left a few minor suggestions. Great work on the new page!

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+[float]
+[[custom-ti-integration]]
+== Enable a custom integration
+
+To enable a custom integration:
+
+. Set up a way to <<ingest-data, ingest data>> into your system.
+. Update the `securitySolution:defaultThreatIndex` <<update-threat-intel-indices, advanced setting>> by adding the appropriate index pattern name after the default {fleet} threat intelligence index pattern (`logs-ti*`), for example, `custom-ti-index*`.
++
+NOTE: Threat intelligence indices aren’t required to be ECS compatible. However, we strongly recommend compatibility if you’d like for your alerts to be enriched with relevant threat indicator information. You can find a list of ECS compliant threat intelligence fields at {ecs-ref}/ecs-threat.html[Threat Fields].
```
Comment (joepeeples) [inline docs/getting-started/threat-intel-integrations.asciidoc:46]:
```suggestion
NOTE: Threat intelligence indices aren’t required to be ECS compatible. However, we strongly recommend compatibility if you’d like for your alerts to be enriched with relevant threat indicator information. You can find a list of ECS-compliant threat intelligence fields at {ecs-ref}/ecs-threat.html[Threat Fields].
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
++
+NOTE: The Threat Intelligence view looks for a `threat.feed.name` field value to define the source name in the *Name* column. If a custom source doesn't have the `threat.feed.name` field, or hasn't defined a `threat.feed.name` field value, it's named *Other*. In addition, dashboards links are not typically not created for sources in the *Other* category unless a source has the `threat.feed.dashboard_id` field defined.
```
Comment (joepeeples) [inline docs/getting-started/threat-intel-integrations.asciidoc:50]: Also, "not typically created" seems kind of vague -- any way we can explain more precisely?
```suggestion
NOTE: The Threat Intelligence view searches for a `threat.feed.name` field value to define the source name in the *Name* column. If a custom source doesn't have the `threat.feed.name` field or hasn't defined a `threat.feed.name` field value, it's named *Other*. In addition, dashboards links are not typically created for sources in the *Other* category unless a source has the `threat.feed.dashboard_id` field defined.
```

**Review by joepeeples — COMMENTED** [review]
This might look like a lot of comments, but a lot of them are just small line-edit suggestions for conciseness. 😄 Hope they're helpful!

Overall, it's great to have a separate page for the threat intel info and procedures. I'd suggest moving even more info to that separate page, and keep trimming down what's included on the Elastic Security UI page. I think that the UI overview page works better when it's just a very general, top-level intro to parts of the Security app, and not a deep dive into anything. Almost all of the other sections on the UI page are just a sentence or two on each part of the app, and then the link to a separate page for more details.

**Review by nkhristinin — APPROVED** [review]
LGTM!

File: docs/getting-started/security-ui.asciidoc [hunk 7]
```diff
@@ -79,19 +79,23 @@ image::images/events-count.png[Shows host and network events on the Overview pag
...
+//The Threat Intelligence view shows the total number of threat indicator events collected by the Threat Intel module. This module ingests, parses, and visualizes data from several open source threat feeds to provide you with relevant and timely threat intelligence. As a security analyst, you can use this data in a variety of ways. For example, you can reference this data while building new threat indicator match rules or triaging alerts. Or, you use it to assess the robustness of your organization's security controls and make suggestions for improvements. Regardless of your goal, the Threat Intelligence view offers you timely and actionable threat intelligence that you can leverage for your, or your organization's, detection and prevention needs.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/getting-started/security-ui.asciidoc:82]: It's not needed and we can always recover this in previous versions of the file. Deleted!

File: docs/getting-started/threat-intel-integrations.asciidoc (hunk 3 above)
Comment (nastasha-solomon) [reply docs/getting-started/threat-intel-integrations.asciidoc:44]: Good point! I'll update these examples.

**Comment by nastasha-solomon** [comment]
@shimonmodi would you mind taking a look at these docs as well? 

@ecezalp and I have been discussing adding in an alternative step for accessing Agent and Filebeat threat intel integrations and could use your perspective on whether that'd be helpful. 

We're also wondering how these integrations should be framed. To elaborate, should threat intel integrations using Agent be referred to as `Elastic Agent threat intelligence integrations` or `Fleet threat intelligence integrations`? Similar question about Filebeat integrations - should they be referred to as `Filebeat threat intel integrations`, `Threat intel module integrations`, or something else?

**Review by ecezalp — APPROVED** [review]
LGTM

File: docs/getting-started/security-ui.asciidoc [hunk 8]
```diff
@@ -79,19 +79,14 @@ image::images/events-count.png[Shows host and network events on the Overview pag
 
 *Threat Intelligence*
 
-The Threat Intelligence view shows the total number of threat indicator events collected by the Threat Intel Filebeat module. This module ingests, parses, and visualizes data from several open source threat feeds to provide you with relevant and timely threat intelligence. As a security analyst, you can use this data in a variety of ways. For example, you can reference this data while building new threat indicator match rules or triaging alerts. Or, you use it to assess the robustness of your organization's security controls and make suggestions for improvements. Regardless of your goal, the Threat Intel Filebeat module offers you timely and actionable threat intelligence that you can leverage for your, or your organization's, detection and prevention needs.
+The Threat Intelligence view on the Overview page gives you a streamlined way to collect threat intelligence data for threat detection and matching.
```
Comment (jmikell821) [inline docs/getting-started/security-ui.asciidoc:82]:
```suggestion
The Threat Intelligence view on the Overview page provides a streamlined way to collect threat intelligence data for threat detection and matching.
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 9]
```diff
@@ -0,0 +1,74 @@
+[[es-threat-intel-integrations]]
+= Enable threat intelligence integrations
+
+The Threat Intelligence view gives you a streamlined way to collect threat intelligence data that you can use for threat detection and matching. Threat intelligence data consists of threat indicators ingested from third-party threat intelligence sources.
```
Comment (jmikell821) [inline docs/getting-started/threat-intel-integrations.asciidoc:4]:
```suggestion
The Threat Intelligence view provides a streamlined way to collect threat intelligence data that you can use for threat detection and matching. Threat intelligence data consists of threat indicators ingested from third-party threat intelligence sources.
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+- If you've chosen a time range that doesn't contain threat indicator event data, you are prompted to choose a different range. Use the the date and time picker in the {security-app} or Kibana to select a new range to analyze.
+- If the {agent} or {filebeat} agent hasn't ingested Threat Intel module data yet, the threat indicator event counts won't load. You can wait for data to be ingested or reach out to your administrator for help resolving this.
+
+[float]
+[[agent-ti-integration]]
+== Enable an {agent} integration
+
+To set up an {agent} integration:
+
+. Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] on the servers you want to monitor.
```
Comment (jmikell821) [inline docs/getting-started/threat-intel-integrations.asciidoc:26]:
```suggestion
. Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] on the hosts you want to monitor.
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+. In the Threat Intelligence view, click *Enable sources*. Scroll down the Integration page and select *Elastic Agent only* so the view only shows {agent} integrations.
```
Comment (jmikell821) [inline docs/getting-started/threat-intel-integrations.asciidoc:27]:
```suggestion
. In the Threat Intelligence view, click *Enable sources* to view the Integrations page. Scroll down and select *Elastic Agent only* to filter by {agent} integrations.
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+== Enable a {filebeat} Threat Intel module integration
+
+To enable a {filebeat} Threat intel integration:
+
+. Set up the {filebeat-ref}/filebeat-installation-configuration.html[{filebeat} agent] and enable the Threat Intel module.
++
+NOTE: For more information about enabling available threat intelligence filesets, refer to {filebeat-ref}/filebeat-module-threatintel.html[Threat Intel module].
+
+. Update the `securitySolution:defaultThreatIndex` <<update-threat-intel-indices, advanced setting>> by adding the appropriate index pattern name after the default {fleet} threat intelligence index pattern (`logs-ti*`):
+.. If you're _only_ using {filebeat} version 8.x, add the appropriate {filebeat} threat intelligence index pattern, for example, `logs-ti*`, `filebeat-8*`.
```
Comment (jmikell821) [inline docs/getting-started/threat-intel-integrations.asciidoc:57]:
```suggestion
.. If you're _only_ using {filebeat} version 8.x, add the appropriate {filebeat} threat intelligence index pattern. For example, `logs-ti*`, `filebeat-8*`.
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+[float]
+[[custom-ti-integration]]
+== Enable a custom integration
+
+To enable a custom integration:
+
+. Set up a way to <<ingest-data, ingest data>> into your system.
+. Update the `securitySolution:defaultThreatIndex` <<update-threat-intel-indices, advanced setting>> by adding the appropriate index pattern name after the default {fleet} threat intelligence index pattern (`logs-ti*`), for example, `logs-ti*`,`custom-ti-index*`.
++
+NOTE: Threat intelligence indices aren’t required to be ECS compatible. However, we strongly recommend compatibility if you’d like for your alerts to be enriched with relevant threat indicator information. You can find a list of ECS-compliant threat intelligence fields at {ecs-ref}/ecs-threat.html[Threat Fields].
```
Comment (jmikell821) [inline docs/getting-started/threat-intel-integrations.asciidoc:70]:
```suggestion
NOTE: Threat intelligence indices aren’t required to be ECS compatible. However, we strongly recommend compatibility if you’d like your alerts to be enriched with relevant threat indicator information. You can find a list of ECS-compliant threat intelligence fields at {ecs-ref}/ecs-threat.html[Threat Fields].
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+
+. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
++
+NOTE: The Threat Intelligence view searches for a `threat.feed.name` field value to define the source name in the *Name* column. If a custom source doesn't have the `threat.feed.name` field, or hasn't defined a `threat.feed.name` field value, it's considered unnamed and labeled as *Other*. Dashboards aren't created for unnamed sources unless the `threat.feed.dashboard_id` field is defined.
```
Comment (jmikell821) [inline docs/getting-started/threat-intel-integrations.asciidoc:74]:
```suggestion
NOTE: The Threat Intelligence view searches for a `threat.feed.name` field value to define the source name in the *Name* column. If a custom source doesn't have the `threat.feed.name` field or hasn't defined a `threat.feed.name` field value, it's considered unnamed and labeled as *Other*. Dashboards aren't created for unnamed sources unless the `threat.feed.dashboard_id` field is defined.
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+image::images/threat-intelligence-view.png[width=90%][height=90%][Shows the Threat Intelligence view on the Overview page]
+
+There are a few scenarios when data won't display in the Threat Intelligence view:
+
+- If you've chosen a time range that doesn't contain threat indicator event data, you are prompted to choose a different range. Use the the date and time picker in the {security-app} or Kibana to select a new range to analyze.
+- If the {agent} or {filebeat} agent hasn't ingested Threat Intel module data yet, the threat indicator event counts won't load. You can wait for data to be ingested or reach out to your administrator for help resolving this.
+
+[float]
+[[agent-ti-integration]]
+== Enable an {agent} integration
```
Comment (jmikell821) [inline docs/getting-started/threat-intel-integrations.asciidoc:22]:
```suggestion
== Configure an {agent} integration
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 16]
```diff
@@ -0,0 +1,74 @@
...
+
+To set up an {agent} integration:
```
Comment (jmikell821) [inline docs/getting-started/threat-intel-integrations.asciidoc:24]: You can omit this intro. 
```suggestion

```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 17]
```diff
@@ -0,0 +1,74 @@
...
+
+* AbuseCH
+* AlienVault OTX
+* Anomali
+* Cybersixgill
+* ThreatQuotient
+* MISP
+
+=========================
+. Select an {agent} integration, and complete the installation steps.
```
Comment (jmikell821) [inline docs/getting-started/threat-intel-integrations.asciidoc:41]:
```suggestion
. Select an {agent} integration, then complete the installation steps.
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 18]
```diff
@@ -0,0 +1,74 @@
...
+NOTE: For more information about enabling available threat intelligence filesets, refer to {filebeat-ref}/filebeat-module-threatintel.html[Threat Intel module].
+
+. Update the `securitySolution:defaultThreatIndex` <<update-threat-intel-indices, advanced setting>> by adding the appropriate index pattern name after the default {fleet} threat intelligence index pattern (`logs-ti*`):
+.. If you're _only_ using {filebeat} version 8.x, add the appropriate {filebeat} threat intelligence index pattern, for example, `logs-ti*`, `filebeat-8*`.
+.. If you're using a previous version of Filebeat _and_ a current one, differentiate between the threat intelligence indices by using unique index pattern names. For example, if you’re using {filebeat} version 7.0.0 and 8.0.0, update the setting to `logs-ti*`,`filebeat-7*`,`filebeat-8*`.
+. Return to the Threat Intelligence view on the Overview page. Refresh the page if indicator data isn't displaying.
+
+[float]
+[[custom-ti-integration]]
+== Enable a custom integration
```
Comment (jmikell821) [inline docs/getting-started/threat-intel-integrations.asciidoc:63]:
```suggestion
== Configure a custom integration
```

File: docs/getting-started/threat-intel-integrations.asciidoc [hunk 19]
```diff
@@ -0,0 +1,74 @@
...
+
+To enable a custom integration:
```
Comment (jmikell821) [inline docs/getting-started/threat-intel-integrations.asciidoc:65]:
```suggestion

```

**Review by jmikell821 — COMMENTED** [review]
Looks like a lot, but most of these are small comments.

**Pull request merged**